src/after-compile.ts

```diff
@@ -246,18 +246,20 @@ function provideDeclarationFilesToWebpack(
     }
 
     const outputFiles = getEmitOutput(instance, filePath);
-    const declarationFile = outputFiles
-      .filter(outputFile => outputFile.name.match(constants.dtsDtsxRegex))
-      .pop();
-    if (declarationFile !== undefined) {
-      const assetPath = path.relative(
-        compilation.compiler.context,
-        declarationFile.name
-      );
-      compilation.assets[assetPath] = {
-        source: () => declarationFile.text,
-        size: () => declarationFile.text.length
-      };
+    const declarationFiles: typescript.OutputFile[] = outputFiles
+      .filter((outputFile: typescript.OutputFile) => outputFile.name.match(constants.dtsDtsxRegex));
```

Comment:
can `(outputFile: typescript.OutputFile)` just be `outputFile`?

Reply:
Yeah, you are correct. I reinstalled VS Code, and it works now. I'm removing these.